R/util-MakeBounds.R

```diff
@@ -16,9 +17,12 @@
 #' @return A data frame.
 #'
 #' @examples
+#'  \dontrun{
```

Comment:
Why `/dontrun`? It feels like this should run fine with the sample data.

Reply:
yeah, i'll delete a bunch of these `dontrun`s. not sure what i was thinking this morning when i added all of them...